src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -26,21 +26,26 @@ When you migrate your users to this model, benefits include:
 
-Requirements and guidelines: 
 
-* You must have the [Owner role](/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model/#roles) to start using the user migration wizard. 
-* If your organization has more than 20 accounts, we recommend **not** using the user migration wizard. **You can disregard this requirement if you're being helped by New Relic representatives.** (Note that this refers to having 20 or more accounts in your [organization](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure/#new-model), not users.)
-* If you frequently [add accounts](/docs/accounts/accounts-billing/account-structure/mastersub-account-structure/#creating) to your organization, we recommend **not** using the wizard yet. Currently, users on the new user model require our assistance to add accounts. 
-* Roles related to user management (ability to add and update users, change user type, create access grants) currently apply for an entire organization and can't be assigned at an account level. If you require account-level user management role assignments, we recommend not using the wizard. 
-* For customers using SAML SSO or SCIM: If you were previously using one of the New Relic-specific Azure, Okta, or OneLogin apps, note that you'll need to use a newer app version titled "New Relic by organization" (the previous one was titled "New Relic by account").
+## Recommendations and considerations [#recommendations]
 
-If you don't meet these requirements but still want to migrate your users in order to take advantage of the new model benefits, talk to your New Relic account representative. 
+We have some recommendations below that apply to organizations who choose to do the user migration on their own. Note that **these considerations won't apply if you're being helped by a New Relic representative**; in that case, your account representative will give you guidance. 
+
+* If your organization has more than 20 accounts, we recommend **not** using the user migration wizard unless you're getting help from a New Relic representative. (Note that this refers to having 20 or more accounts, **not** users.)
+* If you frequently [add accounts](/docs/accounts/accounts-billing/account-structure/mastersub-account-structure/#creating) to your organization, we recommend **not** using the wizard yet. Currently, users on the new user model require New Relic assistance to add accounts. 
+* We have some changes in functionality pertaining to alerts that may affect some customers. [Read about user model impacts.](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#limitations)
+* Roles related to user management (ability to add and update users, change user type, create access grants) currently apply across an organization and can't be assigned at an account level. If you require account-level user management role assignments, we recommend not using the wizard. 
```

Comment:
There are several bullet points that re-iterate "we recommend not using the wizard" so maybe these bullet points could be more clearly and tersely mentioned under a sentence that stated something like "Don't use the wizard under the following circumstances:"